lib/cluster/index.ts

```diff
@@ -842,6 +849,24 @@ class Cluster extends EventEmitter {
           }
         }
 
+        // Assign to each node keys a numeric value to make autopipeline comparison faster.
+        this._groupsIds = {};        
```

Comment:
Maybe Object.create(null) - should be slightly faster?

Reply:
Done!